appveyor.yml

```diff
@@ -36,25 +39,34 @@ cache:
 shallow_clone: true
 
 install:
-  # We want  protoc.exe on PATH.
-  - SET PATH=C:/%RIPPLED_DEPS_PATH%;%PATH%
-
   # Download dependencies if appveyor didn't restore them from the cache.
   # Use 7zip to unzip.
   - ps: |
-        if (-not(Test-Path 'C:/$env:RIPPLED_DEPS_PATH')) {
-            echo "Download from $env:RIPPLED_DEPS_URL"
-            Start-FileDownload "$env:RIPPLED_DEPS_URL"
-            7z x "$($env:RIPPLED_DEPS_PATH).zip" -oC:\ -y > $null
-            if ($LastExitCode -ne 0) { throw "7z failed" }
+        if (-not(Test-Path "C:/$env:RIPPLED_DEPS_PATH")) {
+            $files = @(
+              "$env:RIPPLED_BOOST",
+              "$env:RIPPLED_BOOST_STAGE",
+              "$env:RIPPLED_OPENSSL"
+              )
+            For ($i=0; $i -lt $files.Length; $i++) {
+                $file = $files[$i]
+                $url = "$env:RIPPLED_DEPS_BASE_URL/$file"
+                echo "Download $file from $url"
+                Start-FileDownload "$url"
+                7z x "$file" -o"C:\$env:RIPPLED_DEPS_PATH" -y > $null
+                if ($LastExitCode -ne 0) { throw "7z failed" }
+            }
+        }
+        else
+        {
+            "Dependencies are in cache"
```

Comment:
is that supposed to be an `echo "Dependencies are in cache"` ?

Reply:
No. One of the few tricks I know in Powershell is that a plain string is just written to stdout.

Reply:
Ah, line 54 had an `echo`, wonder if there is some nuance to it. (could you hook calls to `echo`? but not stdout?)

Reply:
It's probably even simpler than that. Different authors at different times, for example. If you think it's confusing, I can make it consistent.

Reply:
I was just curious, I had to test it out myself to see if it worked or not, I wasn't sure if it was supposed to be a comment or if you intended to print to the console.